tsdapilib/tests.py

```diff
@@ -0,0 +1,239 @@
+
+import os
+import pytest
+import shutil
+import time
+
+from datetime import datetime, timedelta
+
+import blake3
+import psycopg2
+
+from tsdapilib.verifier import ContentVerifier
+from tsdapilib.backends import GenericBackend, SQLiteBackend, PostgreSQLBackend
+
+class Resources(object):
+
+    def __init__(self, work_dir: str) -> None:
+        """
+        Creates:
+
+        /tmp/{work_dir}
+        /tmp/{work_dir}/f1
+        /tmp/{work_dir}/d1
+        /tmp/{work_dir}/d1/f2
+        /tmp/{work_dir}/d1/f3
+        /tmp/{work_dir}/d2
+        /tmp/{work_dir}/d2/f4
+
+        """
+        base_dir = '/tmp'
```

Comment:
I was wondering why you manually set this up instead of using `tempfile.TemporaryDirectory`, is it not kept around while Resources is in context?

Reply:
mostly because I've never used tempfile, but I should have a look if it can simplify it

Reply:
I guess it simplifies the class a bit, as in you don't need to care about cleaning up before/after. It's safer in that you don't delete arbitrary trees under the basepath.